### PR TITLE
Fix test-cmd.sh on OSX

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
[test]

**Comment by liggitt** [comment]
@stevekuznetsov PTAL

**Comment by stevekuznetsov** [comment]
LGTM

**Comment by liggitt** [comment]
[merge]

File: hack/test-cmd_util.sh [hunk 1]
```diff
@@ -155,11 +155,11 @@ os::cmd::expect_success_and_text 'if (( 1 )); then echo "hello"; fi' 'hello'
 os::cmd::expect_success_and_text 'echo $(( 1 - 20 ))' '\-19' # we need to escape for grep
 
 # redirects
-os::cmd::expect_failure_and_text 'grep' 'for more information'
+os::cmd::expect_failure_and_text 'grep' '(Usage|usage)'
```
Comment (ncdc) [inline hack/test-cmd_util.sh]: grep -i?

Comment (stevekuznetsov) [inline hack/test-cmd_util.sh]: Passing flags into `os::cmd` text checks isn't supported.

**Pull request merged**